ui/library/less/styles.less

```diff
@@ -24,8 +15,26 @@
 }
 
 .reader__page-overlay__bounding-box-highlighted {
-  background-color: @pdflib-highlight-background;
-  opacity: .2;
+  background-color: @primary-blue;
+  mix-blend-mode: screen;
+
+  &:hover {
+    background-color: @cornflower-blue;
+    mix-blend-mode: screen;
+  }
+
+  &.selected {
+    background-color: @cornflower-blue;
+    mix-blend-mode: screen;
+  }
+}
+
+.reader__page-over__bounding-box-highlighted-border {
+  pointer-events: none;
+  border-bottom: 2px dotted @primary-blue;
+  margin-top: 3px;
+  position: absolute;
+  z-index: @z-index-bbox-overlay;
```

Comment:
Seems like prettier is doing nothing to the `.less` file in this repo. Make sure the rules are lexically ordered.
The `border-bottom` seems to be different to the figma value.
<img width="211" alt="Screen Shot 2022-04-04 at 13 50 40" src="https://user-images.githubusercontent.com/84034381/161635575-cbb42813-41fe-49de-978c-5fc9c72ff9a8.png">

Also we haven't discussed anything about the abbreviations in this project. Some people may not know what do you mean by `bbox` and `btn` (though it is common on other projects). So I think it's better to use the whole name instead (use `boundingbox` for `bbox`).

Reply:
@yensung that is true but since this PR already had a lot of files i would do it separate PR so i wont have more files to begin with

Reply:
If so then better create a ticket for that and append the number here for note.